recipes/newrelic/infrastructure/super-agent/debian.yml

```diff
@@ -582,9 +603,19 @@ install:
           IS_AGENT_INSTALLED=$(sudo ps aux | grep newrelic-super-agent | grep -v grep | wc -l)
           if [ $IS_AGENT_INSTALLED -eq 0 ] ; then
             echo "The newrelic super agent has not started after installing. Please try again later, or see our documentation for installing manually https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/install-configure/install-new-relic" >&2
+
+            if [ {{.IS_SYSTEMCTL}} -gt 0 ]; then
```

Comment:
I moved the vars to the root level so they can be accesible from any task to avoid code duplication. 
regarding removing that i would leave it out of scope for this PR.

Reply:
Fair enough.